core/src/trezor/wire/protocol_common.py

```diff
@@ -58,9 +58,6 @@ async def read(
 
     async def write(self, msg: protobuf.MessageType) -> None: ...
 
-    def write_force(self, msg: protobuf.MessageType) -> Awaitable[None]:
```

Comment:
Note: Removed `write_force`  as it is not used by any code in `main` or in the PR. @mmilata